[0m[[0m[31merror[0m] [0m[0mD:\Projects\FP\kursovaya\src\test\scala\laba3\AdtsTests.scala:35:19: not found: value Success[0m
[0m[[0m[31merror[0m] [0m[0m        val res = Success(4)[0m
[0m[[0m[31merror[0m] [0m[0m                  ^[0m
[0m[[0m[31merror[0m] [0m[0mone error found[0m
